package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "iframe-resizer",
-  "version": "4.2.2",
```

Comment:
Can you please leave this as 4.2.2. My deploy script will up the version number automagically

Reply:
done